packages/fluentui/react-northstar/src/themes/teams/components/Card/cardVariables.ts

```diff
@@ -39,11 +46,20 @@ export interface CardVariables {
 
 export default (siteVars): CardVariables => {
   return {
+    backgroundColor: siteVars.colorScheme.default.background,
+    backgroundColorDisabled: siteVars.colorScheme.default.backgroundDisabled,
+    backgroundColorFocus: siteVars.colorScheme.default.backgroundFocus,
+    backgroundColorHover: siteVars.colorScheme.default.backgroundHover,
+    backgroundColorPressed: siteVars.colorScheme.default.backgroundPressed,
     borderStyle: 'solid',
     borderColor: siteVars.colors.black,
+    borderColorDisabled: siteVars.colorScheme.default.borderDisabled,
+    borderColorHover: siteVars.colors.black,
```

Comment:
This is not safe for theme switching, can you check with design, or if it's not by redline, let's add some color scheme value that will look decent in all themes.

Reply:
I'n still waiting for redlines :)
OK, I'll switch to some other color defined everywhere